lib/registration.pm

```diff
@@ -515,10 +515,7 @@ sub process_scc_register_addons {
                     send_key 'alt-a';
                     last;    # Exit yast scc register, no package need be install
                 }
-                else {
-                    record_soft_failure 'bsc#1040758';
-                    next;    # Yast may popup dependencies or software install dialog, enter determine statement again.
-                }
+                next;
```

Comment:
we still need this "next" ? seems you just remove the record_soft_failure.

Reply:
I just checked the PR for the softfail, we shouldn't with the next if remove the workaround. I will remove it and verify